File: utils/AppUtils.cs
using System;
using Model;
using Model.Common;

namespace Utils
{
    public static class AppUtils
    {
        public static void BeautifulMenu()
        {
            Console.WriteLine("+++++++++++++++++++++++++++++++++++++");
            Console.WriteLine("++wwww+++++++++++++++++++++wwww++++++");
            Console.WriteLine("+++wwww++++++++++++++++++wwww++++++++");
            Console.WriteLine("++++wwww++++wwwwwww++++wwww++++++++++");
            Console.WriteLine("++++++wwww+wwww+wwww+wwww++++++++++++");
            Console.WriteLine("+++++++wwwwww++++wwwwwww+++++++++++++");
            Console.WriteLine("+++++++++++++++++++++++++++++++++++++");
            Console.WriteLine("+++++++++++++++++++++++++++++++++++++");
        }

        public static void Menu()
        {
            Console.WriteLine("Welcome to MyWallet!");
            BeautifulMenu();
            Console.WriteLine("1 - Put object in");
            Console.WriteLine("2 - Put object out");
            Console.WriteLine("3 - Get money balance");
            Console.WriteLine("4 - Get objects in a specific pocket");
            Console.WriteLine("5 - Find object (provide the identifier)");
            Console.WriteLine("6 - Count object [Credit card, visit card, driving card, National Card]");
            Console.WriteLine("7 - Indicate object lost or found");
        }

        public static void ObjectTypeMenu(Wallet wallet, int objectId, int pocketNumber, int type)
        {
            switch (type)
            {
                case 1:
                    Console.Write("Money amount : ");
                    double amount = Convert.ToDouble(Console.ReadLine());
                    wallet.PutObjectIn(pocketNumber, new Money(objectId, amount));
                    break;
                case 2:
                    wallet.PutObjectIn(pocketNumber, new Photo(objectId));
                    break;
                case 3:
                    wallet.PutObjectIn(pocketNumber, new Card(objectId, CardType.CREDIT_CARD));
                    break;
                case 4:
                    wallet.PutObjectIn(pocketNumber, new Card(objectId, CardType.VISIT_CARD));
                    break;
                case 5:
                    wallet.PutObjectIn(pocketNumber, new Card(objectId, CardType.DRIVING_CARD));
                    break;
                case 6:
                    wallet.PutObjectIn(pocketNumber, new Card(objectId, CardType.NI_CARD));
                    break;
                default:
                    Console.WriteLine("Unrecognized object type ");
                    break;
            }
        }

        public static void ObjectCountMenu(Wallet wallet, int type)
        {
            switch (type)
            {
                case 1:
                    Console.WriteLine($"Credit card count : {wallet.CountObject(CardType.CREDIT_CARD)}");
                    break;
                case 2:
                    Console.WriteLine($"Credit card count : {wallet.CountObject(CardType.VISIT_CARD)}");
                    break;
                case 3:
                    Console.WriteLine($"Credit card count : {wallet.CountObject(CardType.DRIVING_CARD)}");
                    break;
                case 4:
                    Console.WriteLine($"Credit card count : {wallet.CountObject(CardType.NI_CARD)}");
                    break;
                default:
                    Console.WriteLine("Unrecognized object type ");
                    break;
            }
        }
    }
}

File: utils/AppUtils.py
from model import Card, Money, Photo
from model import CardType


class AppUtils:
    @staticmethod
    def beautiful_menu():
        print("+++++++++++++++++++++++++++++++++++++")
        print("++wwww+++++++++++++++++++++wwww++++++")
        print("+++wwww++++++++++++++++++wwww++++++++")
        print("++++wwww++++wwwwwww++++wwww++++++++++")
        print("++++++wwww+wwww+wwww+wwww++++++++++++")
        print("+++++++wwwwww++++wwwwwww+++++++++++++")
        print("+++++++++++++++++++++++++++++++++++++")
        print("+++++++++++++++++++++++++++++++++++++")

    @staticmethod
    def menu():
        print("Welcome to MyWallet!")
        AppUtils.beautiful_menu()
        print("1 - Put object in")
        print("2 - Put object out")
        print("3 - Get money balance")
        print("4 - Get objects in a specific pocket")
        print("5 - Find object (provide the identifier)")
        print("6 - Count object [Credit card, visit card, driving card, National Card]")
        print("7 - Indicate object lost or found")

    @staticmethod
    def object_type_menu(wallet, object_id, pocket_number, obj_type):
        if obj_type == 1:
            amount = float(input("Money amount: "))
            wallet.put_object_in(pocket_number, Money.Money(object_id, amount))
        elif obj_type == 2:
            wallet.put_object_in(pocket_number, Photo.Photo(object_id))
        elif obj_type == 3:
            wallet.put_object_in(pocket_number, Card.Card(object_id, CardType.CardType.CREDIT_CARD))
        elif obj_type == 4:
            wallet.put_object_in(pocket_number, Card.Card(object_id, CardType.CardType.VISIT_CARD))
        elif obj_type == 5:
            wallet.put_object_in(pocket_number, Card.Card(object_id, CardType.CardType.DRIVING_CARD))
        elif obj_type == 6:
            wallet.put_object_in(pocket_number, Card.Card(object_id, CardType.CardType.NI_CARD))
        else:
            print("Unrecognized object type")

    @staticmethod
    def object_count_menu(wallet, obj_type):
        if obj_type == 1:
            print(f"Credit card count: {wallet.count_object(CardType.CardType.CREDIT_CARD)}")
        elif obj_type == 2:
            print(f"Visit card count: {wallet.count_object(CardType.CardType.VISIT_CARD)}")
        elif obj_type == 3:
            print(f"Driving card count: {wallet.count_object(CardType.CardType.DRIVING_CARD)}")
        elif obj_type == 4:
            print(f"NI card count: {wallet.count_object(CardType.CardType.NI_CARD)}")
        else:
            print("Unrecognized object type")
